src/main/java/com/gary/backendv2/controller/TutorialController.java

```diff
@@ -38,20 +39,26 @@ public void deleteTutorial(@PathVariable Integer tutorialId) {
         tutorialService.deleteTutorial(tutorialId);
     }
 
-    @PostMapping("/{tutorialId}/{userId}")
-    public void addReviewToTutorial(@PathVariable Integer tutorialId, @PathVariable Integer userId,
+    @PostMapping("/{tutorialId}/{email}")
+    public void addReviewToTutorial(@PathVariable Integer tutorialId, @PathVariable @Email String email,
                                     @RequestBody @Valid ReviewRequest reviewRequest) {
-        tutorialService.addReviewToTutorial(userId,tutorialId,reviewRequest);
+        tutorialService.addReviewToTutorial(email,tutorialId,reviewRequest);
     }
 
-    @DeleteMapping("/{tutorialId}/{userId}/{reviewId}")
-    public void deleteReviewFromTutorial(@PathVariable Integer tutorialId, @PathVariable Integer userId,
+    @DeleteMapping("/{tutorialId}/{email}/{reviewId}")
```

Comment:
Jeżeli usuwamy po id tutoriala to powinno być to wystarczające bez email do identyfikacji